packages/applications/ssr/src/utils/historique/mapToProps/installationAvecDispositifDeStockage/events/mapToInstallationAvecDispositifDeStockageModifiéeTimelineItemsProps.tsx

```diff
@@ -0,0 +1,27 @@
+import { Lauréat } from '@potentiel-domain/projet';
+
+export const mapToinstallationAvecDispositifDeStockageModifiéeTimelineItemsProps = (
+  record: Lauréat.InstallationAvecDispositifDeStockage.InstallationAvecDispositifDeStockageModifiéeEvent,
+) => {
+  const { modifiéeLe, modifiéePar, installationAvecDispositifDeStockage } = record.payload;
+
+  return {
+    date: modifiéeLe,
+    title: (
+      <div>
+        Installation avec dispositif de stockage modifiée par{' '}
+        {<span className="font-semibold">{modifiéePar}</span>}
+      </div>
+    ),
+    content: (
+      <div className="flex flex-col gap-2">
+        <div>
+          Nouvelle valeur pour installation avec dispositif de stockage :{' '}
```

Comment:
Comme c'est un boolean, je ferai plutôt une formulation du type 
```suggestion
          A une installation avec dispositif de stockage : oui / non
```